components/config-mgmt-service/grpcserver/cfg_mgmt.go

```diff
@@ -146,15 +147,39 @@ func (s *CfgMgmtServer) GetCheckInCountsTimeSeries(ctx context.Context,
 
 	startTime, endTime := calculateStartEndCheckInTimeSeries(daysAgo)
 
-	timeseries, err := s.client.GetCheckinCountsTimeSeries(startTime,
+	checkinTimeseries, err := s.client.GetCheckinCountsTimeSeries(startTime,
 		endTime.Add(-time.Millisecond), filters)
+	if err != nil {
+		return &response.CheckInCountsTimeSeries{}, errors.GrpcErrorFromErr(codes.Internal, err)
+	}
+
+	deleteTimeseries, err := s.client.GetDeletedCountsTimeSeries(startTime,
+		endTime.Add(-time.Millisecond), filters)
+	if err != nil {
+		return &response.CheckInCountsTimeSeries{}, errors.GrpcErrorFromErr(codes.Internal, err)
+	}
+
+	createTimeseries, err := s.client.GetCreateCountsTimeSeries(startTime,
+		endTime.Add(-time.Millisecond), filters)
+	if err != nil {
+		return &response.CheckInCountsTimeSeries{}, errors.GrpcErrorFromErr(codes.Internal, err)
+	}
```

Comment:
This uses three requests to elasticsearch to collect all the information for the response.